mysql-test/suite/sys_vars/t/pxc_strict_mode_basic.test

```diff
@@ -0,0 +1,50 @@
+#
+# Test different setting option for pxc-strict-mode
+
+# let's cache the default value first.
+#--let $pxc_strict_mode_saved = `SELECT @@pxc_strict_mode`
+
+--echo # set default value
+--error ER_WRONG_VALUE_FOR_VAR
```

Comment:
I don't understand why changing the value of pxc_strict_mode fails for all cases (even for valid values).  Isn't pxc_strict_mode dynamic?